card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
I think, like the various rails gems, we want to keep all these versions synced, so ideally we'd put them all in one place.